app/graphql/types/mutation_type.rb

```diff
@@ -4,6 +4,7 @@ class MutationType < Types::BaseObject
     field :sign_in, mutation: Mutations::SignIn
 
     field :create_project, mutation: Mutations::CreateProject
+    field :create_comment, mutation: Mutations::CreateComment
     field :update_project, mutation: Mutations::UpdateProject
```

Comment:
I think we should put it smth like:
```suggestion
    field :create_project, mutation: Mutations::CreateProject
    field :update_project, mutation: Mutations::UpdateProject
    
    field :create_comment, mutation: Mutations::CreateComment
```